src/templates/components/imageAndStaticMap/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { MediaImage } from '@/templates/common/mediaImage'
+import { recordEvent } from '@/lib/analytics/recordEvent'
+
+import { MediaImage as FormattedMediaImage } from '@/types/formatted/media'
+
+type ImageAndStaticMapProps = {
+  image: FormattedMediaImage
+  facilityId: string
+}
+
+export const ImageAndStaticMap: React.FC<ImageAndStaticMapProps> = ({
+  image,
+  facilityId,
+}) => {
+  const handleImageClick = () => {
```

Comment:
does this need to exist or can we move it directly to the onClick handler?

`onClick={() => recordEvent()}`